docs/landscape.html

```diff
@@ -759,7 +759,8 @@ <h3><a name="Landscape">Landscape</a></h3>
        </li>
       </ul>
      </li>
-     <li>m8 bits 15..12 : free</li>
+     <li>m8 bits 15..13 : free</li>
+     <li>m8 bit 12 : House is protected from removal by town or AI</li>
```

Comment:
As you only need 1 bit, instead of cutting into 4 contiguous free bits, can you perhaps use bit 5 of m3 instead?